superset/assets/src/explore/components/controls/SubscriberLayer.jsx

```diff
@@ -48,13 +45,10 @@ const propTypes = {
   allowMoreColumns: PropTypes.bool,
   allowColumnSelection: PropTypes.bool,
   useAsModal: PropTypes.bool,
-  actionType: PropTypes.string,
 };
 
 const defaultProps = {
   name: '',
-  operatorType: '',
-  columnType: '',
   overrides: {},
   subscriptionList: [],
   subscribe_columns: [],
```

Comment:
subscribe_columns really required if we have subscriptionList

Reply:
removed subscriptionList